models/organization/team_repo.go

```diff
@@ -81,5 +81,6 @@ func GetTeamsWithAccessToRepo(ctx context.Context, orgID, repoID int64, mode per
 		Join("INNER", "team_repo", "team_repo.team_id = team.id").
 		And("team_repo.org_id = ?", orgID).
 		And("team_repo.repo_id = ?", repoID).
+		OrderBy("name").
```

Comment:
```suggestion
		OrderBy(user_model.GetOrderBy()).
```

?

Reply:
No, the properties of the team structure do not have full names

https://github.com/go-gitea/gitea/blob/f1af2919ddfb31b3f87bb61ee28fc6abce3c49e0/models/organization/team.go#L71